playground/Elasticsearch/Elasticsearch.AppHost/Program.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Projects;
+
+var builder = DistributedApplication.CreateBuilder(args);
+
+var elasticsearch = builder.AddElasticsearch("elasticsearch")
```

Comment:
`Elasticsearch` hurts my eyes, but I guess that's what the product is stylized as.

Reply:
if you mean `Elasticsearch ` and `ElasticSearch`, yes `Elasticsearch` is the correct one but it's a little annoying.